edgecast/waf/custom_rule.go

```diff
@@ -0,0 +1,62 @@
+// Copyright Verizon Media, Licensed under the terms of the Apache 2.0 license. See LICENSE file in project root for terms.
+// Package waf provides an API for managing Web Application Firewall for the EdgeCast CDN.
+// WAF  provides a layer of security between security threats and your external web infrastructure.
+package waf
+
+import (
+	"fmt"
+	"strings"
+	"time"
+)
+
+type CustomRuleSet struct {
+	//Indicates the system-defined ID for the custom rule set.
+	Id string `json:"id"`
+
+	//Indicates the date and time at which the custom rule was last modified.
+	//Syntax:
+	//MM/DD/YYYYhh:mm:ss [AM|PM]
+	LastModifiedDate shortDateTime `json:"last_modified_date"`
+
+	//Indicates the name of the custom rule set.
+	Name string `json:"name"`
+}
+
+// Used to handle value returned for LastModifiedDate to allow for implementation of UnmarshalJSON below
+type shortDateTime struct {
+	string
+}
+
+//Allows for LastModifiedDate field within customRuleSets struct to be of formatted datetime string
```

Comment:
shortDateTime struct and UnmarshalJSON funcs should be removed, they will conflict with Frank's code